fenix/app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -332,6 +356,23 @@ class FirefoxColors(
     // Selected tab
     var layerAccentOpaque by mutableStateOf(layerAccentOpaque)
         private set
+
+    // Status indicator dialog background
```

Comment:
Could we stick to the description used in the [stickersheet](https://www.figma.com/file/pEyGeE4KV5ytYHeXMfLcEr/Mobile-Styles?type=design&node-id=1637-23964&mode=design&t=IjKu9IYFyEcnzFRv-0)? 

I think we should leave it to the Design System team to describe what the variable means. If there is no description probably, then I would just avoid having as well.